.pre-commit-config.yaml

```diff
@@ -20,6 +22,6 @@ repos:
         ]
     - id: mypy 
       name: mypy 
-      entry: mypy
-      language: python
-      types: [python]
\ No newline at end of file
+      entry: pipenv run mypy
+      language: system
+      types: [python]
```

Comment:
This is more of an issue regarding environments... Normally from the terminal, you would be in some virtual environment that has all the dependencies installed, but using a UI tool uses the system context instead of the virtual environment. This is what happens with vscode version control UI, so the pipenv tell it to pre-commit using the pipenv virtual environment